webClient/src/app/browser-panel/browser-panel.component.ts

```diff
@@ -60,15 +61,32 @@ export class BrowserPanelComponent implements OnInit {
 
     list: FileRow[];
 
-    leftPath: string;
+    selectedPath: string;
 
     errorMessages: Message[] = [];
 
     contextSide: FTASide;
     contextTreeNode: TreeNode;
     contextRow: FileRow;
 
-    constructor(private uploader: UploaderService) { }
+    uploadModalVisible: boolean;
+
+    encodings = [
+        {
+            content: 'BINARY',
+            selected: true
+        },
+        {
+            content: 'IBM-1047',
```

Comment:
There is not just one type of EBCDIC, there are many. I suggest that instead of having just one, "IBM-1047", that you instead have a mapping between human-readable selections & the REST API's code-name, and expand your list to include these:

International EBCDIC 1047
German/Austrian EBCDIC 273
Danish/Norwegian EBCDIC 277
Finnish/Swedish EBCDIC 278
Italian EBCDIC 278
Japanese Katakana 290
French EBCDIC 297
Arabic (type 4) EBCDIC 420
Hebrew EBCDIC 424
International EBCDIC 500
Thai EBCDIC 838
Croat/Czech/Polish/Serbian/Slovak EBCDIC 870
Greek EBCDIC 875
Urdu EBCDIC 918
Cyrillic(Russian) EBCDIC 1025
Turkish EBCDIC 1026
Farsi Bilingual EBCDIC 1097
Baltic Multilingual EBCDIC 1112
Devanagari EBCDIC 1137
Chinese Traditional EBCDIC 937
Chinese Simplified EBCDIC 935
Japanese EBCDIC 930
Japanese EBCDIC 931
Japanese EBCDIC 939
Japanese EBCDIC 1390
Japanese EBCDIC 1399
Korean EBCDIC 933

Reply:
What I mean by mapping is that "Japanese EBCDIC 930" likely corresponds to IBM-930 in the actual REST call needed to be executed.